sendgrid/exporter_test.go

```diff
@@ -20,7 +21,11 @@ func TestExporterCollect(t *testing.T) {
 	accountNames := map[string]string{
 		"mockAccount": "mockAPIKey",
 	}
-	exporter := NewExporter(accountNames)
+
+	timeZones := map[string]*time.Location{
+		"mockAccount": time.UTC,
+	}
+	exporter := NewExporter(accountNames, timeZones)
```

Comment:
**issue**: This test case won't compile, because the `NewExporter` function only takes one argument.

Please ensure that all the test cases pass before marking the code for review.